Dangerfile

```diff
@@ -6,3 +6,8 @@ github.dismiss_out_of_range_messages
 warn('PR is classed as Work in Progress') if github.pr_title.include? '[WIP]'
 
 rubocop.lint inline_comment: true, fail_on_inline_comment: true
+
+swiftlint.binary_path = './Pods/SwiftLint/swiftlint'
+# Lint all files to ensure maximum coverage.
+swiftlint.lint_all_files = true
+swiftlint.lint_files
```

Comment:
It's also possible to leave inline comments, but I'm not sure how it would work with the `lint_all_files` option.

That's something I'm keen to try in the future, but decided to leave out of this PR to get it merged faster.